sdk-setup/src/sdk-assistant

```diff
@@ -291,9 +291,11 @@ fi
 
 if [[ $OPT_UPDATE ]]; then
     if [[ $OPT_TYPE == target ]]; then
+        echo "Updating the tooling '$OPT_TOOLING_NAME'..."
         _ sdk-manage tooling refresh "$OPT_TOOLING_NAME" || exit
         _ sdk-manage tooling update "$OPT_TOOLING_NAME" || exit
     fi
+    echo "Updating the $OPT_TYPE '$OPT_TOOLING_NAME'..."
```

Comment:
Shouldn't this be "Updating the $OPT_TYPE '$OPT_NAME'..." ?

Reply:
oops, thanks!